radio/src/pulses/modules_helpers.h

```diff
@@ -299,8 +299,30 @@ inline bool isModuleSBUS(uint8_t moduleIdx)
   return g_model.moduleData[moduleIdx].type == MODULE_TYPE_SBUS;
 }
 
+inline bool isModuleAFHDS3(uint8_t idx)
+{
+  return g_model.moduleData[idx].type == MODULE_TYPE_AFHDS3;
+}
+
+inline bool isModulePxx2(uint8_t idx){
+  if(idx != EXTERNAL_MODULE) return false;
```

Comment:
newline before return false
space between if and (

Reply:
Fixed